src/openrct2/scripting/bindings/entity/ScStaff.cpp

```diff
@@ -438,6 +438,170 @@ namespace OpenRCT2::Scripting
         return static_cast<uint8_t>(animationGroup.frame_offsets.size());
     }
 
+    ScHandyman::ScHandyman(EntityId Id)
+        : ScStaff(Id)
+    {
+    }
+
+    void ScHandyman::Register(duk_context* ctx)
+    {
+        dukglue_set_base_class<ScStaff, ScHandyman>(ctx);
+        dukglue_register_property(ctx, &ScHandyman::lawnsMown_get, nullptr, "lawnsMown");
+        dukglue_register_property(ctx, &ScHandyman::gardensWatered_get, nullptr, "gardensWatered");
+        dukglue_register_property(ctx, &ScHandyman::litterSwept_get, nullptr, "litterSwept");
+        dukglue_register_property(ctx, &ScHandyman::binsEmptied_get, nullptr, "binsEmptied");
+    }
+
+    Staff* ScHandyman::GetHandyman() const
+    {
+        return ::GetEntity<Staff>(_id);
+    }
+
+    DukValue ScHandyman::lawnsMown_get() const
+    {
+        auto& scriptEngine = GetContext()->GetScriptEngine();
+        auto* ctx = scriptEngine.GetContext();
+        auto peep = GetHandyman();
+        if (peep != nullptr && peep->AssignedStaffType == StaffType::Handyman)
+        {
+            duk_push_uint(ctx, peep->StaffLawnsMown);
+        }
+        else
+        {
+            duk_push_undefined(ctx);
```

Comment:
I would prefer `null` instead of `undefined` here (same with the other attributes). This might seem to be the opposite of what I said earlier, but now the situation is different since we have split up the interfaces. It will not show up in the API, but be more consistent with how hidden attributes work in other places (e.g. TileElements).